packages/oid4vci-holder/src/types/IOID4VCIHolder.ts

```diff
@@ -333,18 +334,10 @@ export enum SupportedDidMethodEnum {
 
 export type VerificationResult = {
   result: boolean
-  source: WrappedVerifiableCredential | WrappedVerifiablePresentation
```

Comment:
Please leave existing things in place. They really are needed

Reply:
also not ever used

Reply:
Okay but then we need a fixme/todo

Reply:
added FIXME